### PR TITLE
Improve help message for view, check and build subcommands

### PR DESCRIPTION
**Pull request by jiangliu**
Improve help message for view, check and build subcommands, also fix several panic issues.

### CODE REVIEW

**Review by bergwolf — APPROVED** [review]

File: contrib/nydusify/cmd/nydusify.go [hunk 1]
```diff
@@ -153,65 +179,200 @@ func main() {
...
+				&cli.StringFlag{
+					Name:     "target-suffix",
+					Required: false,
+					Usage:    "Generate the target image reference by adding a suffix to the source image reference, conflicts with --target",
+					EnvVars:  []string{"TARGET_SUFFIX"},
+				},
+				&cli.BoolFlag{
+					Name:     "source-insecure",
+					Required: false,
+					Usage:    "Allow http/insecure network communication for source image reference",
```
Comment (imeoer) [inline contrib/nydusify/cmd/nydusify.go:230]: Should be "Skip verifying server certs for HTTPS source registry".

File: contrib/nydusify/cmd/nydusify.go [hunk 2]
```diff
@@ -153,65 +179,200 @@ func main() {
...
+					EnvVars:  []string{"SOURCE_INSECURE"},
+				},
+				&cli.BoolFlag{
+					Name:     "target-insecure",
+					Required: false,
+					Usage:    "Allow http/insecure network communication for target image reference",
```
Comment (imeoer) [inline contrib/nydusify/cmd/nydusify.go:236]: Ditto.

File: contrib/nydusify/cmd/nydusify.go [hunk 3]
```diff
@@ -153,65 +179,200 @@ func main() {
...
+				&cli.StringFlag{
+					Name:    "build-cache-version",
+					Value:   "v1",
+					Usage:   "Version number to filter cache images",
+					EnvVars: []string{"BUILD_CACHE_VERSION"},
+				},
+				&cli.BoolFlag{
+					Name:     "build-cache-insecure",
+					Required: false,
+					Usage:    "Allow http/insecure communication channel for cache images",
```
Comment (imeoer) [inline contrib/nydusify/cmd/nydusify.go:286]: Ditto.

File: contrib/nydusify/cmd/nydusify.go [hunk 4]
```diff
@@ -153,65 +179,200 @@ func main() {
...
+					Required: false,
+					Usage: "Specify a chunk dict expression for chunk deduplication, " +
+						"for examples: bootstrap:registry:localhost:5000/namespace/app:chunk_dict, bootstrap:local:/path/to/chunk_dict.boot",
+					EnvVars: []string{"CHUNK_DICT"},
+				},
+				&cli.BoolFlag{
+					Name:     "chunk-dict-insecure",
+					Required: false,
+					Value:    false,
+					Usage:    "Allow http/insecure communication channel for chunk dicts",
```
Comment (imeoer) [inline contrib/nydusify/cmd/nydusify.go:310]: Ditto.

File: contrib/nydusify/cmd/nydusify.go [hunk 5]
```diff
@@ -330,36 +480,90 @@ func main() {
...
+				&cli.StringFlag{
+					Name:     "target",
+					Required: true,
+					Usage:    "Target (Nydus) image reference",
+					EnvVars:  []string{"TARGET"},
+				},
+				&cli.BoolFlag{
+					Name:     "source-insecure",
+					Required: false,
+					Usage:    "Allow http/insecure network communication for source image reference",
```
Comment (imeoer) [inline contrib/nydusify/cmd/nydusify.go:500]: Ditto.

File: contrib/nydusify/cmd/nydusify.go [hunk 6]
```diff
@@ -330,36 +480,90 @@ func main() {
...
+					EnvVars:  []string{"SOURCE_INSECURE"},
+				},
+				&cli.BoolFlag{
+					Name:     "target-insecure",
+					Required: false,
+					Usage:    "Allow http/insecure network communication for target image reference",
```
Comment (imeoer) [inline contrib/nydusify/cmd/nydusify.go:506]: Ditto.

File: contrib/nydusify/cmd/nydusify.go [hunk 7]
```diff
@@ -388,34 +592,78 @@ func main() {
...
+				&cli.StringFlag{
+					Name:     "target",
+					Required: true,
+					Usage:    "Target (Nydus) image reference",
+					EnvVars:  []string{"TARGET"},
+				},
+				&cli.BoolFlag{
+					Name:     "target-insecure",
+					Required: false,
+					Usage:    "Allow http/insecure network communication for target image reference",
```
Comment (imeoer) [inline contrib/nydusify/cmd/nydusify.go:608]: Ditto.

**Comment by imeoer** [comment]
Others LGTM!

File: contrib/nydusify/cmd/nydusify.go (hunk 1 above)
Comment (jiangliu) [reply contrib/nydusify/cmd/nydusify.go:230]: All fixed

**Pull request merged**